% $Log: abstract.tex,v $
% Revision 1.1  93/05/14  14:56:25  starflt
% Initial revision
% 
% Revision 1.1  90/05/04  10:41:01  lwvanels
% Initial revision
% 
%
%% The text of your abstract and nothing else (other than comments) goes here.
%% It will be single-spaced and the rest of the text that is supposed to go on
%% the abstract page will be generated by the abstractpage environment.  This
%% file should be \input (not \include 'd) from cover.tex.
The extensive genomic diversity of complex systems, such as the human gut microbiome and the evolution of human cancer, has been revealed with the help of recent advances in DNA sequencing. But we are still at an early stage in understanding this genomic diversity to expand our knowledge in biology and for biomedical applications. Taking the diverse human gut microbiome as an example, little is known about the rapid exchange of antibiotic resistance genes and virulence factors as part of the mobile gene flow between the microbes in the gut.

Understanding such heterogeneous systems often involves studing the nature and behavior of the individual cells that constitute the system and their interactions. However, it is technically challenging to probe the genomic material of cells, the smallest unit of life and amplify single genomes for sequencing. Current single-cell technologies require complex instrumentation and the data quality is often confounded by biased genome coverage and chimera artifacts. We address these challenges with a new single-cell technology paradigm to make high-quality low-input genomic research accessible to scientists. 

We developed hydrogel-based \textit{virtual microfluidics} as a simple and robust platform for the compartmentalization of nucleic acid amplification reactions. We applied whole genome amplification (WGA) to purified DNA molecules, cultured bacterial cells, human gut microbiome samples, and human cell lines in the \textit{virtual microfluidics} system. We demonstrated whole-genome sequencing of single-cell WGA products with excellent coverage uniformity and markedly reduced chimerism compared with traditional methods. Together with colleagues, we applied single-cell sequencing to identify horizontally transferred genes between the microbes in the gut and revealed human activities' selective pressure in shaping the mobile gene pools. 

Altogether, we expect \textit{virtual microfluidics} will find application as a low-cost digital assay platform and as a high-throughput platform for single-cell sample preparation. This thesis work made a significant improvement in making high-quality low-input genomic research accessible to scientists in microbiology and oncology. 
% put quantified impact, like how much throughput, how low cost 
 %The genetic diversities of single cells in microbial systems and human cancer have been revealed in both basic sencice and increasingly impact and function and mechansims 
%In microbiome, single-cell genome diversity reveals higher frequenting transfer
% Human cell, brain development and tumor revolution treatment 
% Although, itâ€™s well recognized, the technology to study is limiting 
% But we are still at the early stage of leveraging the genomic diversity to 